src/client/src/orangehrmPerformancePlugin/pages/AddReview.vue

```diff
@@ -0,0 +1,214 @@
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text class="orangehrm-main-title" tag="h6">
+        {{ $t('performance.add_review') }}
+      </oxd-text>
+      <oxd-divider />
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <employee-autocomplete
+                v-model="review.employee"
+                :rules="rules.employee"
+                required
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <supervisor-autocomplete
+                v-model="review.supervisorReviewer"
+                :rules="rules.supervisorReviewer"
+                :subordinate="review.employee"
+                required
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <date-input
+                v-model="review.startDate"
+                :label="$t('performance.review_period_start_date')"
+                :rules="rules.startDate"
+                required
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <date-input
+                v-model="review.endDate"
+                :label="$t('performance.review_period_end_date')"
+                :rules="rules.endDate"
+                required
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <date-input
+                v-model="review.dueDate"
+                :label="$t('performance.due_date')"
+                :rules="rules.dueDate"
+                required
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <div class="orangehrm-button-row">
+            <oxd-button
+              display-type="ghost"
+              :label="$t('general.cancel')"
+              type="button"
+              @click="onCancel"
+            />
+            <oxd-button
+              display-type="ghost"
+              :label="$t('general.save')"
+              type="submit"
```

Comment:
Changed to fix https://github.com/orangehrm/orangehrm/pull/1292/files/a0e8e2c6064e60e70e94441187cdfdadee41f935#r881173828 @RajithaKumara